liquibase-core/src/test/java/liquibase/verify/saved_state/compareGeneratedSqlWithExpectedSqlForMinimalChangesets/createView/db2i.sql

```diff
@@ -1,4 +0,0 @@
--- Database: db2i
```

Comment:
Why was this test deleted? Is it invalid, somehow? Same question for the MaxDB test.

Reply:
@XDelphiGrl Nathan and I were troubleshooting an issue I was getting in 4.18 (4.19?) that he wasn't getting in whichever slightly earlier version he was on. Clearing out some of the tests seemed to be part of the solution. Honestly, I don't really understand what the issue was, but Nathan would know. Might be fixed by now anyway?

Reply:
There were likely changes to these files based on some of the changed docs (probably the 'example' fixed up) which caused the tests to fail.

We deleted the files to get past them failing, but they should be re-generated and the updated versions committed by running the tests on a developer's machine. Then we can see what it's thinking is different.